File: beginner217/b.py
s1 = input()
s2 = input()
s3 = input()
ans = ""
li = ["ABC" , "ARC" , "AGC" , "AHC"]
li_2 = [s1, s2, s3]

for elm in li:
    # print("--------------------------")
    # print(elm)
    if elm not in li_2:
        # print("ififif")
        ans = elm
        break;

print(elm)

File: beginner217/b.cpp
#include <bits/stdc++.h>
#define rep(i,n) for (int i = 0;i < (n); ++i)
using namespace std;
using ll = long long;
using P = pair<int, int>;

int main() { 
  string s1, s2, s3;
  cin >> s1 >> s2 >> s3;
  string ans = "";
  bool res = true;
  vector<string> vec1 = {s1, s2, s3};
  vector<string> vec2 = {"ABC", "ARC", "AGC", "AHC"};
  rep(i,3){
    rep(j, 4){
      if(vec1[i] == vec2[j]){

        continue;
      } else {
        ans = vec1[i];
        break;
      }
    }
  }
  cout << ans << endl;
  return 0;
}